yaml_test_runner/skip.yml

```diff
@@ -41,6 +41,10 @@
       - Bad window deprecated interval
     free/search/230_interval_query.yml:
       - "*"
+    free/search/260_sort_mixed.yml:
```

Comment:
@Xtansia please hold on with skipping, I will update the test case (on OS side) to pass the Rust test suites

Reply:
Update: https://github.com/opensearch-project/OpenSearch/pull/7193 + https://github.com/opensearch-project/opensearch-rs/pull/150 should fix the issue